include/linux/sched.h

```diff
@@ -34,6 +34,7 @@
 #include <linux/rseq.h>
 #include <linux/seqlock.h>
 #include <linux/kcsan.h>
+#include "linux/df-detection.h"
```

Comment:
Please put this header where applicable.
Note that we're using angle brackets for the headers residing in include/linux.